### ISSUE TITLE
 Vuforia Studio Documentation 1.0.2

### ISSUE THREAD

**Issue by patrickscheper** [ptc-ar-sharing/vuforiastudio#2]
Right now we're in the process of extending the repository with more documentation and different examples. 

Related to examples we hope to add a few related how to make rest calls and how to query the touch gestures on a mobile phone.

With the better written TML widget documentation we'll write a small article for the next release how to integrate and create shaders (OpenGL).

And as last @Unfaehig is extending the documentation with the properties of the first few 2D widgets.

## tl;dr
- New example projects
- 2D widget documentation
